### PR TITLE
included required for all forms and note, fix resume size

### LINKED ISSUE #722: Add Dietary Restrictions Label
image

### LINKED ISSUE #723: Make Required
image

### LINKED ISSUE #725: Add Required Note
image

- add a note at the end of the description that shows that the * means

### LINKED ISSUE #726: Make Required
image

### LINKED ISSUE #727: Make Required
image

### LINKED ISSUE #728: Make Required
image

### LINKED ISSUE #729: Make Required
image
image

- Add operations as an affiliation
- Change to software engineering

### PR DESCRIPTION
**Pull request by rfairooz**
Screenshot 2023-10-08 at 4 18 30 PM
Screenshot 2023-10-08 at 4 18 05 PM

### CODE REVIEW

File: src/components/dynamic/Radio.jsx [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+}) => {
   const handleClick = (option) => {
     setUser({ ...user, [field]: option });
   };
 
   return (
     <div className="flex flex-col">
-      <p className="mb-1 font-semibold">{text}</p>
+      <p className="mb-1 font-semibold">
+        {text}{" "}
```
Comment (shahdivyank) [inline src/components/dynamic/Radio.jsx]: remove the space?!

File: src/components/dynamic/Radio.jsx [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+        <span className="text-hackathon-green-300">{required && " *"}</span>
```
Comment (shahdivyank) [inline src/components/dynamic/Radio.jsx]: move the required out so like 
```
required &&  <span className="text-hackathon-green-300">*</span>
```

File: src/components/dynamic/Radio.jsx (hunk 1 above)
Comment (rfairooz) [reply src/components/dynamic/Radio.jsx]: ahhh I think npm run format added that by itself. lemme remove

File: src/components/dynamic/Radio.jsx (hunk 2 above)
Comment (rfairooz) [reply src/components/dynamic/Radio.jsx]: ok fs

File: src/components/dynamic/forms/Form.jsx [hunk 3]
```diff
@@ -88,7 +98,12 @@ const Form = ({ object, setObject, header, fields, onSubmit }) => {
                 )}
                 {field.input === "checkboxes" && (
                   <>
-                    <p className="mb-1 font-semibold">{field.text}</p>
+                    <p className="mb-1 font-semibold">
+                      {field.text}
+                      {field.text && field.required && (
```
Comment (shahdivyank) [inline src/components/dynamic/forms/Form.jsx]:
```suggestion
                      {field.required && (
```

File: src/components/dynamic/forms/Upload.jsx [hunk 4]
```diff
@@ -31,7 +31,10 @@ const Upload = ({ field, user, setUser, text, maxSize, types }) => {
 
   return (
     <div className="flex flex-col">
-      <p className="mb-0 font-semibold">{text}</p>
+      <p className="mb-0 font-semibold">
+        {text}{" "}
+        <span className="text-hackathon-green-300">{required && " *"}</span>
```
Comment (shahdivyank) [inline src/components/dynamic/forms/Upload.jsx]: extract

**Review by shahdivyank — APPROVED** [review]

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**